VimeoNetworking/VimeoNetworking/Request+Comment.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Request+Comment.swift
+//  Pods
+//
+//  Created by King, Gavin on 10/20/16.
+//
```

Comment:
Thats already merged right? I'm going to hold off on adding the headers until after the merge to avoid conflicts.